lib/networks.ts

```diff
@@ -35,7 +87,9 @@ import {
     explorerUrl: 'https://explorer.xai-chain.net',
     isArbitrum: true,
     isCustom: true,
+    blockTime: 10,
```

Comment:
This should be `0.25`. We added `blockTime` to `L2Network` too, to make it more consistent with `L1Network`.